RabbitMQ.Stream.Client/AMQP/Properties.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Buffers;
+using System.Runtime.InteropServices;
+
+namespace RabbitMQ.Stream.Client.AMQP
+{
+    public class Properties : IWritable
```

Comment:
I would unit test this class. Same comment I did earlier for AmqpWireFormatterWriter applies here.